packages/pcd/eddsa-pcd/src/util/util.ts

```diff
@@ -9,17 +24,50 @@ export function newEdDSAPrivateKey(): string {
 }
 
 /**
- * Compares two EdDSA public keys for equality.
+ * Compares two {@link EdDSAPublicKey}s for equality.
  */
 export function isEqualEdDSAPublicKey(
   a: EdDSAPublicKey,
   b: EdDSAPublicKey
 ): boolean {
-  return a[0] === b[0] && a[1] === b[1];
+  if (typeof a === "string" && typeof b === "string") {
```

Comment:
It might be cleaner to normalize both keys into a canonical form (e.g. call publicKeyToPoint) then compare in a single form.  It's hard for me to be sure this function is covering all possible cases.